pkg/inputs/snmp/metadata/poll.go

```diff
@@ -62,6 +62,13 @@ func NewPoller(server *gosnmp.GoSNMP, gconf *kt.SnmpGlobalConfig, conf *kt.SnmpD
 				}
 			}
 		}
+
+		// See if there's a device comment for this profile.
+		dev := profile.GetDeviceSysComment(conf.OID)
+		if dev != "" {
+			log.Infof("Adding model tag %s", dev)
```

Comment:
Will this get "noisy"?  Should we move it to debug?

Reply:
came here to say this 😄

Reply:
Its only done once at device start, but happy to move to debug as this probably isn't info the user cares about.